### PR TITLE
Redirect Landing Page to Admin Login

### PR DESCRIPTION
**Pull request by nirojtogglecorp**
There is no Landing page yet. So , redirecting to admin login.

### CODE REVIEW

File: mis/urls.py [hunk 1]
```diff
@@ -13,15 +13,16 @@
...
 from django.conf.urls.static import static
+from django.contrib import admin
+from django.urls import path, include
+from django.views.generic.base import RedirectView
 
 urlpatterns = [
     path('admin/leave_request/request/', include('leave_request.urls')),
     path('admin/', admin.site.urls),
-    # path('', admin.site.urls),
+    path('', RedirectView.as_view(url='/admin/login/?next=/admin/', permanent=True)),
```
Comment (thenav56) [inline mis/urls.py:25]: It's better to use reverse instead of URL. Could you look if it's available?

Comment (annshress) [reply mis/urls.py:25]: Or remove `admin` url.

Comment (nirojtogglecorp) [reply mis/urls.py]: Used path('', admin.site.urls, name='login')

Comment (thenav56) [reply mis/urls.py]: @nirojtogglecorp 
@annshress suggested this change.
```python
urlpatterns = [
    path('admin/leave_request/request/', include('leave_request.urls')),
    path('', admin.site.urls),
```

Comment (nirojtogglecorp) [reply mis/urls.py]: Worked as suggested.

File: leave_request/views.py [hunk 2]
```diff
@@ -57,5 +57,5 @@ def reject(request):
             text_content = strip_tags(html_content)
             SendEmail.send_mail(subject, text_content, leave_request_exist.verified_by.email,
                                 leave_request_exist.employee.email, html_content)
-            return redirect('/admin/leave_request/request')
+            return redirect('/leave_request/request')
```
Comment (thenav56) [inline leave_request/views.py:60]: Let's use reverse instead of raw url.

Comment (nirojtogglecorp) [reply leave_request/views.py]: Used 
return redirect(reverse('admin:leave_request_leaverequest_changelist'))

**Review by annshress — APPROVED** [review]

**Review by thenav56 — APPROVED** [review]

**Pull request merged**